openreview/venue/webfield/programChairsWebfield.js

```diff
@@ -1,42 +1,61 @@
 // Webfield component
+const automaticAssignment = domain.content.automatic_reviewer_assignment.value
```

Comment:
I'm still a little nervous about not using `?` for accessing these values, because if there is a JS error when running this code the UI can't display anything, just a vague error message. So I would just code any logic in here very defensively.

Reply:
oops sorry, let me add that..

Reply:
Done!